### PR TITLE
force usage of entities for response definition

### PR DESCRIPTION
**Pull request by LeFnord**

### CODE REVIEW

File: spec/swagger_v2/api_swagger_v2_param_type_body_spec.rb [hunk 1]
```diff
@@ -109,43 +109,43 @@ def app
...
-      expect(subject['definitions']['putRequestResponseItem']).to eql({
-        "description" => "put in body with entity",
-        "type"=>"object",
-        "properties"=>{
-          "id"=>{"type"=>"integer", "format"=>"int32", "readOnly"=>true},
-          "name"=>{"type"=>"string", "description"=>"name"}}
-      })
-    end
-  end
+  # describe 'entity given' do
```
Comment (wleeper) [inline spec/swagger_v2/api_swagger_v2_param_type_body_spec.rb]: Is this big comment block intentional or where you going to come back to this?

Comment (LeFnord) [inline spec/swagger_v2/api_swagger_v2_param_type_body_spec.rb]: good point, forgot to remove the comment 😉

**Comment by wleeper** [comment]
Looks like it works well for my case.  

Did you have a use case where it needed to work like it did before this change?

**Comment by LeFnord** [comment]
@wleeper there was no use case for that, I only though it would be nice to have

that's why it is always a good idea, that more then one person have an eye on …

many thanks for it 😄

**Pull request merged**